specification/resources/resource-manager/Microsoft.Authorization/stable/2021-06-01/policyAssignments.json

```diff
@@ -0,0 +1,846 @@
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the policy assignment.",
+          "readOnly": true
+        },
+        "sku": {
+          "$ref": "#/definitions/PolicySku",
+          "description": "The policy sku. This property is optional, obsolete, and will be ignored."
+        },
+        "location": {
+          "type": "string",
+          "description": "The location of the policy assignment. Only required when utilizing managed identity."
+        },
+        "identity": {
+          "$ref": "#/definitions/Identity",
+          "description": "The managed identity associated with the policy assignment."
+        }
+      },
+      "description": "The policy assignment.",
+      "x-ms-azure-resource": true
+    },
+    "PolicyAssignmentListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PolicyAssignment"
+          },
+          "description": "An array of policy assignments."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to use for getting the next set of results."
+        }
+      },
+      "description": "List of policy assignments."
+    },
+    "Identity": {
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The principal ID of the resource identity.  This property will only be provided for a system assigned identity"
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant ID of the resource identity.  This property will only be provided for a system assigned identity"
+        },
+        "type": {
+          "type": "string",
+          "description": "The identity type. This is the only required field when adding a system or user assigned identity to a resource.",
+          "enum": [
+            "SystemAssigned",
+            "UserAssigned",
+            "None"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceIdentityType",
+            "modelAsString": false,
+            "values": [
+              {
+                "value": "SystemAssigned",
+                "description": "Indicates that a system assigned identity is associated with the resource."
+              },
+              {
+                "value": "UserAssigned",
+                "description": "Indicates that a system assigned identity is associated with the resource."
+              },
+              {
+                "value": "None",
+                "description": "Indicates that no identity is associated with the resource or that the existing identity should be removed."
+              }
+            ]
+          }
+        },
+        "userAssignedIdentities": {
+          "type": "object",
+          "additionalProperties": {
```

Comment:
was referencing how compute did their swagger which doesn't exactly apply here.  Will fix this.